src/main/org/apache/tools/ant/taskdefs/optional/junitlauncher/AbstractJUnitResultFormatter.java

```diff
@@ -43,7 +43,7 @@
 /**
  * Contains some common behaviour that's used by our internal {@link TestResultFormatter}s
  */
-abstract class AbstractJUnitResultFormatter implements TestResultFormatter {
+public abstract class AbstractJUnitResultFormatter implements TestResultFormatter {
```

Comment:
It is impossible to have abstract class protected, so it is marked as public. The methods are protected, not public.